index.html

```diff
@@ -2891,6 +2896,66 @@ <h3 id="applying">
         </ol>
       </section>
     </section>
+      <section>
+        <h3>
+          <code title="">iarc_rating_id</code> member
```

Comment:
>I @kenchris was wondering about that "store" part as well. I though it was supposed to work across stores.

AFAIK, the IARC ID is per-app and can be used across stores.  

During the process of submitting an app to a store, an app developer has a choice of entering their IARC ID or getting a new one.

>This means pinging the IARC servers for validation, no? 
> Is it just a GET request? Do we need any special HTTP headers?

In the confirmation email there is a permalink: 
>A permanent record of this certificate may be accessed here.
"here" is hyperlinked.